packages/cli/src/commands/run.ts

```diff
@@ -37,3 +50,46 @@ export const run = populateWithCommonOptions(new Command("run"))
 			}
 		}
 	});
+
+export const validateCmd = populateWithCommonOptions(new Command("validate"))
```

Comment:
I think this can be a separate file right? Run and validate don't rely on a shared function.